README.md

```diff
@@ -18,17 +18,32 @@ For more info, see:
 
 ## Development
 
-1. `npm install`
-2. `./node_modules/web-ext/bin/web-ext run -s src/`
+To run this extension:
+
+1. Open Firefox and load the `about:debugging` page. Click 
+   [Load Temporary Add-on](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Temporary_Installation_in_Firefox)
+   and select the `manifest.json` file within the folder of an example extension.
+   Here is a [video](https://www.youtube.com/watch?v=cer9EUKegG4)
+   that demonstrates how to do this.
+2. Install the
+   [web-ext](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Getting_started_with_web-ext)
+   tool. At the command line, open the example extension's folder and type
+   `web-ext run -s src/`. This launches Firefox and installs the extension automatically.
+   This tool provides some additional development features, such as
+   [automatic reloading](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Getting_started_with_web-ext#Automatic_extension_reloading).
```

Comment:
Could you break these out into two seperate lists? And have the WebExtensions directions first? Directions are easier to follow along when in step/list format. 

Something like: 

```
### Running Locally

#### via WebExtensions API (web-ext)

- Install dependencies  `$ npm install`...

#### via about:debugging in Firefox

- Load `about:debugging` page...

Reply:
Good idea :)